kafka/producer/kafka.py

```diff
@@ -400,7 +400,10 @@ def close(self, timeout=None):
             log.info('Kafka producer closed')
             return
         if timeout is None:
-            timeout = 999999999
+            try:
+                timeout = threading.TIMEOUT_MAX
```

Comment:
and do we also need to assert that a user-supplied timeout is <= threading.TIMEOUT_MAX if on python 3.3+ ?